### ISSUE TITLE
Add apache requirements to README

### ISSUE THREAD

**Issue by cortho** [owncloud/oauth2#110]
It took me quite a while finding out, that both mod_rewrite and mod_headers are required to have oauth2 run on apache.

For me is is not sufficient, that this information _might_ be found somewhere in the _server_ docs (see https://github.com/owncloud/oauth2/issues/49#issuecomment-311763331).

**Comment by SamuAlfageme** [comment]
@cortho indeed; getting this info at first sight both on the app's README and its marketplace page will help more admins getting it right without wasting time. Thanks for the tip!

Also ref. this issue for discussion: https://github.com/owncloud/core/issues/30042.

**Comment by settermjd** [comment]
Are any specific modules required for NGINX?

**Comment by settermjd** [comment]
While NGINX isn't officially supported, as it has a lot of community interest and usage, it'd be handy to know and to document.

**Comment by settermjd** [comment]
I've added a note about `mod_headers` being mandatory for OAuth2 access in `installation/source_installation.html`, in the admin manual in recent changes in #3464.
